omf-python/src/attribute.rs

```diff
@@ -150,3 +156,30 @@ impl PyAttributeDataColor {
         }
     }
 }
+
+#[pyclass(name = "AttributeDataMappedTexture")]
+pub struct PyAttributeDataMappedTexture(pub AttributeData);
+#[pymethods]
+impl PyAttributeDataMappedTexture {
+    #[getter]
```

Comment:
Maybe this shouldn't be a getter() as its copying the image which could be quite large (I'm assuming this how it works), not like copying a scalar or a short array.

(I think we need to do a another pass over how we use getters in other places in the library)

Reply:
I think maybe the image data does not get  read in until the reader is called, it looks like Array's contain the file name. So the copying an Array should be relatively cheap.

Reply:
Agree on the use of [#getter] though, I've used it to be consistent, not for any well thought out reason.